tests/target/string_punctuation.rs

```diff
@@ -0,0 +1,13 @@
+fn main() {
+    println!("ThisIsAReallyLongStringWithNoSpaces.It_should_prefer_to_break_onpunctuation:\
+              Likethisssssssssssss");
+    format!("{}__{}__{}ItShouldOnlyNoticeSemicolonsPeriodsColonsAndCommas{}{}",
```

Comment:
THis string ought to be long enough to break, in order for it to be a useful test